### PR TITLE
Ability to increast the retryamount for lavalink

### PR DESCRIPTION
**Pull request by DarrenOfficial**
**Please describe the changes this PR makes and why it should be merged:**
This pr add a way so that if the lavalink restart it auto reconnects and add upto unlimited reconnect retry amount.

**Status and versioning classification:**

### CODE REVIEW

**Comment by joeyk710** [comment]
Looks good to me

**Review by SudhanPlayz — APPROVED** [review]

File: botconfig.js [hunk 1]
```diff
@@ -22,7 +22,9 @@ module.exports = {
     host: "lava.link",
     port: 80,
     pass: "youshallnotpass", 
-    secure: false, // Set this to true if you're self-hosting lavalink on replit.
+    secure: false, // Set this to true if you're hosting your own lavalink on replit.
+    retrydelay: 15000, // Delay for reconnect in ms.
```
Comment (SudhanPlayz) [inline botconfig.js:26]:
```suggestion
    retryDelay: 15000, // Delay for reconnect in ms.
```

File: structures/DiscordMusicBot.js [hunk 2]
```diff
@@ -137,6 +137,8 @@ class DiscordMusicBot extends Client {
           port: this.botconfig.Lavalink.port,
           password: this.botconfig.Lavalink.pass,
           secure: this.botconfig.Lavalink.secure,
+          retryDelay: this.botconfig.Lavalink.retrydelay,
```
Comment (SudhanPlayz) [inline structures/DiscordMusicBot.js:140]:
```suggestion
          retryDelay: this.botconfig.Lavalink.retryDelay,
```

File: structures/DiscordMusicBot.js [hunk 3]
```diff
@@ -137,6 +137,8 @@ class DiscordMusicBot extends Client {
...
+          retryAmount: this.botconfig.Lavalink.retryamount,
```
Comment (SudhanPlayz) [inline structures/DiscordMusicBot.js:141]:
```suggestion
          retryAmount: this.botconfig.Lavalink.retryAmount,
```

File: botconfig.js [hunk 4]
```diff
@@ -22,7 +22,9 @@ module.exports = {
...
+    retryamount: 10000000, // Retry amout if the lavalink is dead and/or restarting.
```
Comment (SudhanPlayz) [inline botconfig.js:27]:
```suggestion
    retryAmount: 10000000, // Retry amout if the lavalink is dead and/or restarting.
```

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Change

**Comment by DarrenOfficial** [comment]
Fixed 👍🏼

**Review by SudhanPlayz — APPROVED** [review]

**Comment by SudhanPlayz** [comment]
Fix confilcts

**Comment by DarrenOfficial** [comment]
fixed.

**Pull request closed without merging**